android/modules/ui/src/java/ti/modules/titanium/ui/_2DMatrixProxy.java

```diff
@@ -7,11 +7,21 @@
 package ti.modules.titanium.ui;
 
 import org.appcelerator.kroll.annotations.Kroll;
+import org.appcelerator.kroll.common.Log;
 import org.appcelerator.titanium.view.Ti2DMatrix;
 
 @Kroll.proxy(creatableInModule = UIModule.class, name = "2DMatrix")
 public class _2DMatrixProxy extends Ti2DMatrix
 {
+	private static final String TAG = "2DMatrix";
+
+	public _2DMatrixProxy()
+	{
+		super();
+
+		Log.w(TAG, "Ti.UI.2DMatrix DEPRECATED in 8.0.0, in favor of Ti.UI.Matrix3D");
```

Comment:
`Matrix2D`

Reply:
Good thing you still have a watchful eye over this!